client/landing/stepper/declarative-flow/internals/hooks/use-step-navigation-with-tracking/index.ts

```diff
@@ -35,6 +35,13 @@ export const useStepNavigationWithTracking = ( {
 		};
 	}, [] );
 
+	const stepData = useSelect(
```

Comment:
Is it better to return previous step from the store, rather than all the step data? Just thinking about the performance implications...

Reply:
I tried that, but the whole `stepData` object can be null, so I had to check it first before accessing `previousStep`.